### DISCUSSION TITLE
AXLOG("Test") write more as I need: "`?/[2024-03-15 10:00:23.505][PID:37fc][TID:52c4]Test`"

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#1745] [Q&A] [answered]
`AXLOG("Test")`
shows: `?/[2024-03-15 10:00:23.505][PID:37fc][TID:52c4]Test`

I want only:
 `Test`
 
 Which setting is needed?

**Comment by rh101** [comment, marked as answer]
> `AXLOG("Test")`
> shows: `?/[2024-03-15 10:00:23.505][PID:37fc][TID:52c4]Test`

You must be calling `ax::setLogFmtFlag` somewhere in your own code, otherwise the extra info wouldn't be there. So, either delete the call, or use `setLogFmtFlag(LogFmtFlag::Null);`

Also, which platform are you testing this on?

**Reply by aismann** [reply]
Windows 11

**Reply by halx99** [reply]
The cpp-tests enable LogFmtFlag::Full on AppDelegate, just for example. by default the axmol set it to `Null`
